QuizApp/App/Protocols/CoordinatorProtocol.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+protocol CoordinatorProtocol {
+
+    @discardableResult func start() -> UIViewController
```

Comment:
Coordinator should have functions that start with `show` and describe what screen should be presented.

Reply:
Reworked it.